MOV-AI/frontend-npm-lib-react
# FP-1906 - Not authorized screen should allow redirect to login page

https://movai.atlassian.net/browse/FP-1906

To test this development in other apps:
`npm install @mov-ai/mov-fe-lib-react@0.0.1906-2`

- Login with user without permissions - Not Authorized message should appear with a button to go back to login screen
- Login with user with permissions - should login normally
